packages/cursorless-engine/src/util/getPartialTargetDescriptors.ts

```diff
@@ -23,6 +23,9 @@ export function getPartialTargetDescriptors(
     case "replace":
     case "editNew":
       return getPartialTargetDescriptorsFromDestination(action.destination);
+    case "parsed":
+      // FIXME: The args might actually be targets; not sure how to handle
+      return [];
```

Comment:
ok this is annoying. It's causing pre-commit failures because we are now ignoring the args, because we don't know if they're targets. So when it tries to extract targeted marks, it doesn't see them. We could have a hack to try to see if they're targets, but it feels pretty wrong